Cargo.toml

```diff
@@ -15,12 +15,17 @@ harness = false
 name = "stats"
 
 [dependencies]
-erased-serde = "0.3"
-futures = "0.3.5"
 iobuffer = "0.2"
-serde = {version = "1.0", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
-tokio = {version = "0.2.22", features = ["rt-core", "stream", "time"] }
+tokio = { version = "1", features = ["rt-multi-thread", "time"] }
```

Comment:
I wonder if ideally we'd hide this behind a feature flag - if you aren't using the itnerval logging it's a big hit to your depndency tree.